Assets/TriggerTaskMode.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class TriggerTaskMode : MonoBehaviour
+{
+
+    [SerializeField]
+    private string shootingScene, extinguishingScene, craneOperatingScene;
```

Comment:
comment je variabelen (shootingScene, extinguishingScene, craneOperatingScene), wat voor gegevens slaan deze variabelen op?

Bijvoorbeeld: "// Name of Tool scenes"